server/views/pages/overview/partials/educationAndTrainingTab/_educationAndQualificationsHistory.njk

```diff
@@ -171,13 +179,18 @@ never happen.
               <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
               <h3 class="govuk-heading-s" data-qa="other-qualifications">Other qualifications and education history</h3>
-              <p class="govuk-body" data-qa="add-education-history">
+             <p class="govuk-body" data-qa="add-education-history">
                 To add education history, including vocational qualifications,
-                <a href="/prisoners/{{ prisonerSummary.prisonNumber }}/create-induction/hoping-to-work-on-release" class="govuk-link" data-qa="link-to-create-induction">
+                {% if hasEditAuthority %}
```

Comment:
I'm not sure about this .... 🤔  .... as you have it, if the user does not have edit authority, they'll see text that says:

> To add education history, including vocational qualifications, create a learning and work progress plan

IE. they'll see the exact same text as the user who does have edit access, but there will be no link in it. 

We'd be telling the read-only user to create a learning and work progress plan .... but they cant because they are read only.

I would have thought the if block would be better placed around the <p> tag, so if the user is a read-only user, they dont see anything at all here.